lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -321,7 +321,7 @@ def add_pods_library
         # @return [void]
         #
         def add_copy_resources_script_phase
-          native_targets.each do |native_target|
+          native_targets_to_embed_in.each do |native_target|
```

Comment:
This mirrors https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/installer/user_project_integrator/target_integrator.rb#L359